frontend/src/Page.js

```diff
@@ -104,7 +109,7 @@ function Page({backend}) {
     <Container className="screen">
       <Row>
         <Col md={2} className="sources">
-          <DocumentsCards docs={sourcesOfSourceMetadata} byRow={1} />
+          <DocumentsCards docs={sourcesOfSourceMetadata} byRow={1}/>
```

Comment:
Please don't add lines or files unrelated with your contribution.
This makes integration harder (with more conflicts) and git blame less reliable.